### PR TITLE
Fix the #31 issue for ssh connction failed when deploy

### PR DESCRIPTION
**Pull request by iflamed**
This may be is not best way. Because it can't get the real response from the server. Like below.

```shell
Permission denied (publickey).
```

### CODE REVIEW

File: app/Commands/DeployProject.php [hunk 1]
```diff
@@ -247,39 +247,45 @@ private function runStep(DeployStep $step)
...
+                    $process = new Process($this->sshCommand($server, $script, $user));
+                    $process->setTimeout(null);
 
-                $output = '';
-                $process->run(function ($type, $output_line) use (&$output, &$log) {
-                    if ($type == Process::ERR) {
-                        $output .= $this->logError($output_line);
-                    } else {
-                        $output .= $this->logSuccess($output_line);
-                    }
```
Comment (REBELinBLUE) [inline app/Commands/DeployProject.php]: PHPCS: Whitespace found at end of line

**Comment by iflamed** [comment]
I have fixed the phpcs problem. But the ci is still failed. Why?

**Comment by REBELinBLUE** [comment]
I'll merge this in later today, out of interest what SSH server are you using?

I still have not managed to recreate this problem, I've tried with a vagrant VM but thought vagrant might be doing something funny so I spun up a new linode and still can't get it to happen (setting .ssh to 600 it does fail to connect but I get the deploy correctly being marked as failed)

**Comment by iflamed** [comment]
All code is running on cent os 7.0 . The target deploy server is the same machine. 
But in different user.  Php-fpm is running at apache. The queue is listened by  user liubing. And the deployed user is www. 
Below is php version.
```shell
[liubing@develop ~]$ php -v
PHP 5.6.9 (cli) (built: May 15 2015 09:31:38) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies
```

**Pull request merged**